docs/getting_started/index.md

```diff
@@ -0,0 +1,16 @@
+# Getting Started
+
+Welcome to the **Getting Started** guide for Taipy. Taipy is composed of two components that enable anyone to create entire applications:
+
+- **Graphical User Interface builder** (Taipy GUI): allows any Python developer to create a complex and interactive GUI.
+
+- **Scenario Management** (Taipy Core): implements a modern backend for any data-driven application based on your business case.
+Three Getting Started are available. 
```

Comment:
What's the use of this line?

Reply:
To reintroduce Taipy Core and GUI briefly to redirect them to the correct Getting Started